src/scenes/Partnerships/PartnershipForm/PartnershipForm.tsx

```diff
@@ -153,6 +154,11 @@ export const PartnershipForm = <T extends PartnershipFormValues>({
                 </SecuredField>
               </>
             )}
+            {values.partnership?.partnerLookupItem || partnership ? (
```

Comment:
Shouldn't need this condition

Reply:
Though maybe we should hide the switch if the partnership is already primary. We don't allow that on the backend so it would be better just to hide it here.

Reply:
By the way, we need this switch while creating partnership, right?

Reply:
Yeah, so show if no partnership or partnership.primary is false.